challenges/challenge1.html

```diff
@@ -111,6 +111,13 @@ <h1>Terri Fricker</h1>
             <p><q>Stop trying to make a point and focus on making a difference.</q> - Scott Martin</p>
             </div>
 
+            <!-- avatar for Mister Nobody -->
+            <div class="camperCell col-md-5 col-lg-3" id="misterybodon">
+            <h1>Mister Nobody</h1>
+            <img src="https://i.imgur.com/ws5rzqB.jpg"
+               alt="Avatar of Nobody">
+            <p><q cite="https://www.gutenberg.org/files/45315/45315-h/45315-h.htm">The nakedness of woman is the work of God.</q><br>&dagger; William Blake &dagger;</p>
```

Comment:
William Blake is a cool dude!
I just think that content on our page should be more PG. What do you think?